.github/workflows/dotnet.yml

```diff
@@ -29,7 +29,10 @@ jobs:
     - uses: nuget/setup-nuget@b2bc17b761a1d88cab755a776c7922eb26eefbfa # tag=v1
       with:
         nuget-version: '5.x'
-    - run: nuget restore lowkey-vault-example-dotnet.sln
+    - name: NuGet Restore .NET Framework project
+      run: nuget restore dotnet-framework/lowkey-vault-example-dotnet.sln
+    - name: NuGet Restore .NET Core project
+      run: nuget restore dotnet-core/lowkey-vault-example-dotnet.sln
```

Comment:
Thank you for making the changes! I think this line should be referring to `dotnet-core/lowkey-vault-example-dotnet-core.sln`, possibly copy-paste issue. Could you please update it?

Reply:
Yep, you're absolutely right. Sorry, I should have caught this one earlier.